### PR TITLE
Mac os readme

### PR DESCRIPTION
**Pull request by potyl**
Add a section for macOS to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,19 @@ brew link qt --force
...
 make
 ```
+
+
+# Execute
+
+## Linux
+
+```
+build/hakchi-gui
```
Comment (ghoost82) [inline README.md:42]: this should be ./build/hakchi-gui

Comment (potyl) [reply README.md:42]: `build/hakchi-gui` without a leading dot works. A leading dot is needed only when the command lies in the current folder.

This is so that the shell can tell a part between a command that could be in the path or in the current directory. For instance if we type in `foobar` the shell would not know where to look. Should it look in the PATH or in the current folder? The current folder could be a security risk (any one could put their own `ls` and users will be in for a treat).

If a command has a slash anywhere there's no confusion for the shell. A leading slash means an absolute path. Staring with a dot from here, if it starts with .. it goes up to the parent folder and with a tilde (~) to the users account or another user account.

TL;DR if paths have at least a slash all is fine, this is why we use `./foo` to tell the shell that it is in the current folder
